### PR TITLE
Add content for fatigue and defence

### PR DESCRIPTION
**Pull request by Lambosaurus**

### CODE REVIEW

**Review by LouderthanThunder — CHANGES_REQUESTED** [review]
Do we not want to tie fatigue to class/race?

**Comment by Lambosaurus** [comment]
@LouderthanThunder I thought we discussed not tying fatigue to classes. Let feats and strength define fatigue.

File: systems/stats.md [hunk 1]
```diff
@@ -37,11 +37,18 @@ Will is primarially used for [divine magic](spells.md#Divine-magic)
 
 
 ## Defence
-This is a stub
+Defence represents how capable a character is at protecting themselves from foes. This might be using quick footwork or heavy armor plating.
+
+Base defence is [finesse](#Finesse) plus the bonuses of any worn [armor](items.md#Armor).
+- Wearing [heavy armor](items.md#Heavy-armor) removes your [finesse](#Finesse) bonus.
+- Defence may never be negative.
```
Comment (Hoppelite) [inline systems/stats.md:44]: Bold claims

File: scripts/stats.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import random
```
Comment (Hoppelite) [inline scripts/stats.py:1]: I think we should think about moving these into appropriate classes as we get more scripts

Comment (Hoppelite) [reply scripts/stats.py:1]: I'm Invisioning we will eventually have factories and stuff, maybe.

**Review by Hoppelite — APPROVED** [review]

File: scripts/stats.py (hunk 2 above)
Comment (Lambosaurus) [reply scripts/stats.py:1]: I agree. Ideally there will be a character class that generates the appropriate properties, and has methods for attacking. I have chosen not to do this until it is needed.

**Pull request merged**